myrestapi/books.js

```diff
@@ -1,3 +1,75 @@
+<<<<<<< HEAD
```

Comment:
You need to delete this otherwise this file won't be a syntacticly valid js file. Looks like a mishap resolving git conflicts.